crates/containerd-shim-wasm/src/sandbox/shim/local.rs

```diff
@@ -387,7 +387,31 @@ impl<T: Instance + Sync + Send, E: EventSender> Task for Local<T, E> {
     #[cfg_attr(feature = "tracing", tracing::instrument(parent = tracing::Span::current(), skip_all, level = "Info"))]
     fn wait(&self, _: &TtrpcContext, req: WaitRequest) -> TtrpcResult<WaitResponse> {
         debug!("wait: {:?}", req);
-        Ok(self.task_wait(req)?)
+
+        #[cfg(feature = "opentelemetry")]
+        {
+            use tracing::{span, Level, Span};
+
+            let (tx, rx) = std::sync::mpsc::channel();
+            // Start a thread to export interval span for long wait
+            let parent_span = Span::current();
+            let _ = thread::spawn(move || {
+                while rx.recv_timeout(Duration::from_secs(60)).is_err() {
+                    let current_span =
```

Comment:
I think in order to make sure each span is sent we need to call `exit()` on the previous span?

Reply:
when `_enter` goes out of scope here, span automatically exits

Reply:
Oh yeah that's nice, but then when does `_enter` go out of scope here? My intuition says on the next loop iteration.

Reply:
IIUC this goes out of scope immediately. Does it generate a tiny span every 60 seconds? Is that what we want? Or would it be better a span taking the whole 60 seconds?

Reply:
Ah yes it generates tiny spans. I agree a 60s span makes more sense 👍

Reply:
Done

Reply:
Is this current change loog good to y'all?

Reply:
LGTM.
I wonder however if we can achieve this spawning a task instead of a thread. 
Also, I don't fully understand why we need this change.

Reply:
It was something that came up in another project for long-running requests.
Basically the span can't be exported until it exits.

Reply:
I think in this case though, if we don't exit the parent span first then we'll have an orphan span until the parent exits.

Reply:
getting back to this PR, is there anything you wanna change or are you good with merging as is? @cpuguy83

Reply:
It may be good to exit the parent span, but honestly this  is probably fine as is.